dockerfiles/base/scripts/base/commands/cmd_destroy.sh

```diff
@@ -75,3 +75,18 @@ directory_is_empty() {
     return 0
   fi
 }
+
+pre_cmd_destroy() {
+  if get_command_help; then
```

Comment:
do we need to define help inside a pre_'command' ? with a check on `if get_command_help`

we could have commands named `cmd_destroy_help` or `help_cmd_destroy` and in the execution of the lifecycle, it the help command is invoked then it invokes the help command

it will simplify the code really need for the init stuff vs help stuff